tests/test_config.py

```diff
@@ -118,8 +124,11 @@ class CustomJSONEncoder(JSONEncoder):
         assert config.jwt_in_query_string is True
         assert config.jwt_in_cookies is True
         assert config.jwt_in_headers is False
+        assert config.jwt_in_json is False
```

Comment:
For the sake of checking all the config stuff here, could you add json to the JWT_TOKEN_LOCATION in the override configs and make that this is set to True here?